crates/rust-project-goals-cli/src/cfp.rs

```diff
@@ -0,0 +1,533 @@
+        if !dry_run {
+            println!("Updated existing 'Next goal period ({})' section in src/README.md", capitalized_timeframe);
+        } else {
+            println!("Would update existing 'Next goal period ({})' section in src/README.md", capitalized_timeframe);
+        }
+    } else if Regex::new(r"## Next goal period(?:\s*\([^\)]*\))").unwrap().is_match(&content) {
+        if !dry_run {
+            println!("Updated existing 'Next goal period' section in src/README.md");
+        } else {
+            println!("Would update existing 'Next goal period' section in src/README.md");
+        }
+    } else if Regex::new(r"## Current goal period(?:\s*\([^\)]*\))").unwrap().is_match(&content) {
+        if !dry_run {
+            println!("Added new 'Next goal period' section after 'Current goal period' in src/README.md");
+        } else {
+            println!("Would add new 'Next goal period' section after 'Current goal period' in src/README.md");
+        }
+    } else {
+        if !dry_run {
+            println!("Added new 'Next goal period' section to src/README.md");
+        } else {
+            println!("Would add new 'Next goal period' section to src/README.md");
+        }
+    }
+    
+    // Write the updated content back to README.md
+    if !dry_run {
+        fs::write(readme_path, new_content)
+            .with_context(|| format!("Failed to write to src/README.md"))?;
+    }
+    
+    Ok(())
+}
+
+#[cfg(test)]
+mod tests {
+    use super::text_processing::*;
+
+    #[test]
+    fn test_process_template_content() {
+        // Test basic placeholder replacement
+        let content = "# YYYYHN Goals\n\nThis is for YYYY and HN.";
+        let result = process_template_content(content, "2026H1", "2026h1");
+        assert_eq!(result, "# 2026H1 Goals\n\nThis is for 2026 and H1.");
+        
+        // Test note removal
+        let content_with_notes = "# YYYYHN Goals\n\n> **NOTE:** This is a note that should be removed.\n> More note content.\n\nThis should stay.";
+        let result = process_template_content(content_with_notes, "2026H1", "2026h1");
+        assert_eq!(result, "# 2026H1 Goals\n\nThis should stay.");
+        
+        // Test that other blockquotes are preserved
+        let content_with_blockquote = "# YYYYHN Goals\n\n> This is a regular blockquote that should stay.\n\n> **NOTE:** This is a note that should be removed.\n> More note content.\n\nThis should stay.";
+        let result = process_template_content(content_with_blockquote, "2026H1", "2026h1");
+        assert_eq!(result, "# 2026H1 Goals\n\n> This is a regular blockquote that should stay.\n\nThis should stay.");
+    }
+    
+    #[test]
+    fn test_process_summary_content_no_existing_section() {
+        // Test adding a new section when no timeframe sections exist
+        let content = "# Summary\n\n[Introduction](./README.md)\n";
+        let result = process_summary_content(content, "2026h1", "2026h1");
```

Comment:
To me this, and in the other tests that follow, should be 

```suggestion
        let result = process_summary_content(content, "2026H1", "2026h1");
```

like the first test did.